cylc/flow/network/schema.py

```diff
@@ -562,6 +580,26 @@ def resolve_json_dump(root, info, **args):
     return json.loads(field)
 
 
+def resolve_mapping_to_list(root, info, **args):
+    field = getattr(root, to_snake_case(info.field_name), {}) or {}
+    mapping = {
+        key: field[key]
+        for key in args.get('sort_order', [])
+        if key in field}
+    mapping.update(field)
+    satisfied = args.get('satisfied')
+    elements = sort_elements(
+        [
+            val
+            for val in mapping.values()
+            if satisfied is None or val.satisfied is satisfied
```

Comment:
So `satisfied` is always `None` or `True` or `False`?

Reply:
That's determined by us.. Argument not used, then it's None, otherwise, the user has to set it to True or False (we wan't unset to be distinguishable from True/False, as None will mean include both in the result)

Reply:
What I was getting at, if it can be anything but those 3 values, you have to use `val.satisfied == satisfied` (not `is`).

Reply:
Just to check myself:
```python
>>> foo = "true"
>>> bar = "".join(["t", "r", "u", "e"])
>>> foo == bar
True
>>> foo is bar
False
>>> 
```

Reply:
The line about it fetches the argument with `get`
```
   satisfied = args.get('satisfied')
```
If it's not set then `get` will return `None`, and if it is set then GraphQL will fail validation on the Query/Subscription as the field is defined as Boolean.

So it can only be those 3.
And `val.satisfied` is defined as Boolean in the Protobuf definition, so we are covered at both ends.